lib/components/private/Field/Field.tsx

```diff
@@ -124,16 +125,22 @@ export const Field = ({
         />
       ) : null}
 
-      <Box position="relative">
-        <BackgroundProvider value={fieldBackground}>
+      <BackgroundProvider value={fieldBackground}>
+        <Box
+          position="relative"
+          background={fieldBackground}
+          borderRadius="standard"
+          className={secondaryIcon ? styles.secondaryIconSpace : undefined}
+        >
           {children(
             overlays,
             {
               id,
               name,
               background: fieldBackground,
```

Comment:
Do we still need this?

Reply:
Yes, in case the implementing field needs to render a layer with the using background colour. Felt safer that way.